framework/src/controls/PIDTransientControl.C

```diff
@@ -147,7 +151,15 @@ PIDTransientControl::execute()
     if (_dt > 0)
       value += _Kder * delta / _dt;
 
-    value = std::min(std::max(_minimum_output_value, value), _maximum_output_value);
+    // If the maxmimum rate of change by the pid is fixed
+    if ((isParamSetByUser("maximum_change_rate")))
```

Comment:
use the max change rate instead to detect if the parameter is passed (if != default_value)